nucypher/policy/conditions/evm.py

```diff
@@ -329,6 +352,35 @@ def __init__(
         self.function_abi = function_abi
         self.contract_function = self._get_unbound_contract_function()
 
+        # call to super must be at the end for proper validation
+        super().__init__(condition_type=condition_type, method=method, *args, **kwargs)
+
+    def _validate_method(self, method):
+        return method
```

Comment:
Maybe a comment here making explicit that we're declining to raise?

Reply:
I assume you mean a comment here why no check is performed on method - this simply overrides the validation done by the RPCCondition superclass that restricts RPC methods.